autoload/neomake/makers/ft/haskell.vim

```diff
@@ -1,5 +1,6 @@
 
 function! neomake#makers#ft#haskell#MakerAvailable(command)
+    let s:stack_is_executable = 0
```

Comment:
Please init this outside of the function, and use -1 initially.
This way it gets reset when resourcing the file.